src/content/docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial.mdx

```diff
@@ -0,0 +1,204 @@
+    	{name: "HOST3", vertexClass: HOST, definingAttributes: [{key: "host/name", value: "HOST3"}, {key: "availability-zone", value: "us-west-2c"}, {key: "region", value: "us-west-2"}]}]) 
+  {
+    result
+  }
+}
+```
+
+### Create edges [#edges]
+
+This mutation is used to create one or more edges that represent relationships between one vertex to another vertex.
+
+The NerdGraph call below makes use of these fields: 
+
+* `accountId`: Your New Relic account ID.
+* `fromVertexName`:  The name of the vertex you’re defining the edge for.
+* `toVertexName`: The name of the vertex at which you’re ending the edge.
+* `directed`: Optional. A boolean flag that describes how you want to connect one vertex to another vertex. True means the edge can traverse only one direction, indicating a one-way relationship. False means the edge can traverse in both directions, indicating a two-way relationship. By default, directed = `true`. 
+
+Example call: 
+
+```
+mutation {
+  aiTopologyCollectorCreateEdges(
+    accountId: <var>NEW_RELIC_ACCOUNT_ID</var>, 
+   edges: 
+    [
+      {directed: true, fromVertexName: "ServiceA", toVertexName: "ServiceB"},
+      {directed: true, fromVertexName: "ServiceB", toVertexName: "ServiceC"},
+      {directed: false, fromVertexName: "ServiceA", toVertexName: "HOST1"},
+      {directed: false, fromVertexName: "ServiceA", toVertexName: "HOST2"},
+      {directed: false, fromVertexName: "ServiceA", toVertexName: "HOST3"},
+      {directed: false, fromVertexName: "ServiceB", toVertexName: "HOST1"},
+      {directed: false, fromVertexName: "ServiceB", toVertexName: "HOST2"},
+      {directed: false, fromVertexName: "ServiceB", toVertexName: "HOST3"},
+      {directed: false, fromVertexName: "ServiceC", toVertexName: "HOST1"},
+      {directed: false, fromVertexName: "ServiceC", toVertexName: "HOST2"},
+      {directed: false, fromVertexName: "ServiceC", toVertexName: "HOST3"},      
+    ]) {
+    result
+  }
+}
+```
+
+### Delete vertices [#delete-vertices]
+
+This mutation deletes vertices in your topology graph. Note that deleting a vertex deletes all edges connected to it. 
+
+The NerdGraph call below uses these fields:
+* `accountId`: Your New Relic account ID.
+* `vertexNames`: A list of vertex names you want to delete.
+
+Example call: 
+
+```
+mutation {
+  aiTopologyCollectorDeleteVertices(
+    accountId: <var>NEW_RELIC_ACCOUNT_ID</var>, 
+    vertexNames: ["ServiceA", "ServiceB", "ServiceC", "HOST1", "HOST2", "HOST3"]) 
+  {
+    result
+  }
+}
+```
+
+### Delete edges [#delete-edges]
+
+This mutation deletes edges connecting vertices in your topology graph. 
+
+The NerdGraph call below uses these fields:
+
+* `accountId`: Your New Relic account ID.
+* `edgeIds`: A list of edge IDs you want to delete.
+
+Example call: 
+
+```
+mutation {
+  aiTopologyCollectorDeleteEdges(
+    accountId: <var>NEW_RELIC_ACCOUNT_ID</var>, 
+    edgeIds: ["d8a7971b-575d-42e9-aa13-43a50c5a7d10", "0da5cb92-0428-4890-992b-2823d037cb5e"]
```

Comment:
These are fake IDs, yeah?

Reply:
it sholudn't matter to have real ones, don't think. i'll make sure.